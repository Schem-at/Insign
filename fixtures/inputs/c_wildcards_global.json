[
  { "pos": [0, 64, 0], "text": "@cpu.unit1=rc([0,0,0],[2,2,2])\n#cpu.*:logic.clock_hz=4" },
  { "pos": [10, 64, 10], "text": "@cpu.unit2=rc([5,0,0],[7,2,2])\n#$global:io.bus_width=8" }
]
