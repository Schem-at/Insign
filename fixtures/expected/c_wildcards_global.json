{
  "$global": {
    "metadata": {
      "io.bus_width": 8
    }
  },
  "cpu.*": {
    "metadata": {
      "logic.clock_hz": 4
    }
  },
  "cpu.unit1": {
    "bounding_boxes": [
      [
        [
          0,
          64,
          0
        ],
        [
          2,
          66,
          2
        ]
      ]
    ],
    "metadata": {
      "logic.clock_hz": 4
    }
  },
  "cpu.unit2": {
    "bounding_boxes": [
      [
        [
          15,
          64,
          10
        ],
        [
          17,
          66,
          12
        ]
      ]
    ],
    "metadata": {
      "logic.clock_hz": 4
    }
  }
}
